### PR TITLE
Adding travis and semantic-release

### PR DESCRIPTION
**Pull request by roggervalf**
Travis is a CI that could automate processes when push a commit. Semantic-release allow you to generate change-logs and versioning automatically by formatted commits. 
https://www.conventionalcommits.org/en/v1.0.0-beta.4/
I think this could be very helpful.

### CODE REVIEW

**Comment by muratsu** [comment]
Hey Rogger, thanks for the pull request!!! I really love this change but we're already using circleci. It seems redundant to use multiple CI tools for a project. It would be amazing if we can port this to circleci or move everything to GitHub (Github Actions) and don't use any 3rd party dependencies. Thoughts?

**Comment by roggervalf** [comment]
Hey @muratsu, I search for circleci:
https://github.com/semantic-release/semantic-release/blob/master/docs/recipes/circleci-workflows.md
I was working with semantic-release in some of my project and travis, but I guess this commit should be work on circleci.

**Comment by muratsu** [comment]
We definitely want this but it's a bit involved. Do you want to merge this together over the weekend or sometime next week?

Feel free to book some time on my calendar:
https://calendly.com/muratsutunc

File: .circleci/config.yml [hunk 1]
```diff
@@ -14,7 +14,9 @@ jobs:
             - run: yarn install
```
Comment (roggervalf) [inline .circleci/config.yml:14]: delete this line

**Review by roggervalf — COMMENTED** [review]
little comments

**Pull request merged**